.github/workflows/trigger-pr.yaml

```
@@ -0,0 +1,11 @@
+name: Run CI
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+on: [push, pull_request]
```

trigger-pr should only need `pull_request`?

I suppose `ci.yaml` will run twice on push with this configuration?
1. in `trigger-cicd.yaml`
2. in `trigger-pr.yaml`

```suggestion
on: [pull_request]
```